### DISCUSSION TITLE
Binding to class properties

### DISCUSSION THREAD

**Discussion by amn-sharma** [Textualize/textual#4482] [Q&A]
Hi All, 
I am new to textual but not Python. I have mainly developed Windows .net applications, and in Windows apps, we can bind the class properties together to textbox fields or other controls. Is there a similar concept here in textual? 

I am in love with the concept so far and was trying to see if I could start a new personal project with event-driven design.

Thanks

**Comment by davep** [comment]
Perhaps [something like this](https://textual.textualize.io/guide/reactivity/#data-binding)?
